flow/layers/display_list_layer_unittests.cc

```diff
@@ -0,0 +1,161 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#define FML_USED_ON_EMBEDDER
+
+#include "flutter/flow/layers/display_list_layer.h"
+
+#include "flutter/flow/testing/diff_context_test.h"
+#include "flutter/fml/macros.h"
+#include "flutter/testing/mock_canvas.h"
+
+#ifndef SUPPORT_FRACTIONAL_TRANSLATION
+#include "flutter/flow/raster_cache.h"
+#endif
+
+namespace flutter {
+namespace testing {
+
+using DisplayListLayerTest = LayerTest;
+
+#ifndef NDEBUG
```

Comment:
Why is this run in debug only?

Reply:
As per all of the other layer unittests which only support the EXPECT_DEATH tests on debug? Or at least that was my interpretation of the same ifdef in all of the other unit test files. Looking around it looks like there are some similar ifdefs in the other files that wrap even non-Death tests and the final DEATH test here which is outside the ifdef as it was in the picture layer tests that I cut and pasted from.

Reply:
I pushed a commit that moved the 2 EXPECT_DEATH tests in picture/dl_layer_unittests inside the DEBUG ifdef.